framework/audio/internal/src.h

```diff
@@ -0,0 +1,68 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+#ifndef MU_AUDIO_SRC_H
+#define MU_AUDIO_SRC_H
+
+#include <vector>
+
+namespace mu {
+namespace audio {
+class SRC
```

Comment:
Can you explain, please, what is the purpose of this class? And it should to inherit some interface

Reply:
It is a sample rate convertor. It contains math of conversion and methods for do it.
It does not need any interface and used only in internal classes.

Reply:
1. Need to rename this class to SampleRateConvertor
2. I think, it will be too hard to write UT on users of this class because all users have a strong dependency on this class. This class stored its state and contains much business logic

Reply:
Need to rename this class to SampleRateConvertor - +1

Reply:
renamed